include/ginkgo/core/base/math.hpp

```diff
@@ -913,13 +927,7 @@ GKO_INLINE GKO_ATTRIBUTES constexpr xstd::enable_if_t<is_complex_s<T>::value,
                                                       remove_complex<T>>
 abs(const T &x)
 {
-#ifdef CL_SYCL_LANGUAGE_VERSION
-    // FIXME: This implementation is due to a DPC++ issue:
-    // plain `sqrt` call evaluates to `std::sqrt` which fails on GPUs
-    return cl::sycl::sqrt(squared_norm(x));
-#else
     return sqrt(squared_norm(x));
```

Comment:
I am not sure absolute array will call this or not due to namespace issue from dpcpp (sycl).
Also, if the sqrt is not the `std::sqrt` from `kernels::dpcpp`, it will give type error.
I write the above only as note because absolute_array test seems to be passed correctly

Reply:
Yes that's a good point, my expectation is now things are fixed and the `sqrt` is actually the `std::sqrt` we specified above. If you have any issue in practice it's good to keep in mind that this may have to change.